operator/pkg/operator.go

```diff
@@ -332,7 +334,25 @@ func (o *Operator) ProcessNewBatchLogV3(newBatchLog *servicemanager.ContractAlig
 	return nil
 }
 
+func (o *Operator) IsVerifierValid(blacklisted_verifiers_bitmap *big.Int, verifierId common.ProvingSystemId) bool {
+	verifierIdInt := uint64(verifierId)
+	bit := blacklisted_verifiers_bitmap.Uint64() & (1 << verifierIdInt)
```

Comment:
uint8

Reply:
Fixed in [ee0d97d](https://github.com/yetanotherco/aligned_layer/pull/1090/commits/ee0d97d123d0d7a144a19b35c935b4ba4305ff8a)